src/main/java/org/spongepowered/api/item/merchant/TradeOffer.java

```diff
@@ -122,12 +122,29 @@ static Builder builder() {
     boolean hasExpired();
 
     /**
-     * Gets whether this trade offer will grant experience upon usage or not.
+     * Gets whether this trade offer will grant experience to the customer upon usage or not.
      *
-     * @return True if using this trade offer will grant experience
+     * @return True if using this trade offer will grant experience to the customer
      */
     boolean doesGrantExperience();
 
+    /**
+     * Gets the amount of experience granted to the merchant when this trade offer is used.
+     *
+     * @return The experience to be granted to the merchant
+     */
+    int getMerchantExperience();
```

Comment:
@gabizou Do you have a better name of this? It's the experience granted to the merchant when a trade offer is used.

Reply:
I'd say it's fine. It's the experience granted to the merchant on use. just maybe `getExperienceGrantedToMerchantOnUse` but that's verbose.